lib/ImageEditor.js

```diff
@@ -59,7 +64,11 @@ class ImageEditor {
    * will point to the image in the cache path. Remember to delete the
    * cropped image from the cache path when you are done with it.
    */
-  static cropImage(uri: string, cropData: ImageCropData): Promise<string> {
+  static cropImage(
+    uri: string,
+    cropData: ImageCropData,
+    useExternalCache?: boolean,
+  ): Promise<string> {
```

Comment:
We can drop `useExternalCache` now, it is not used

Reply:
Fixed :)